frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/common-template-validations/common-templates-validations.ts

```diff
@@ -45,15 +37,24 @@ export const getTemplateValidations = (
       ? iUserTemplates.find((template) => iGetName(template) === userTemplateName)
       : null;
 
+  if (iUserTemplate) {
+    const validations = getValidationsFromTemplates(List([iUserTemplate]));
+    return getFieldValidations(validations, IDToJsonPath[fieldId]);
```

Comment:
why are we using id for the lookup? We can just use `VMSettingsField` enum

Reply:
? it's a variable of type `VMSettingsRenderableField` , what am I missing here ?

Reply:
the type is correct, but the usage is not. Why are we calling `getFieldId` ?

Reply:
Nice !, didn't notice that , will fix.

Reply:
fixed